EmpleoDotNet.Core/Domain/JobOpportunity.cs

```diff
@@ -75,16 +75,20 @@ public class JobOpportunity : EntityBase
 
         public bool IsActive { get; set; } = true;
 
+        public int? JobOpportunityLocationId { get; set; }
```

Comment:
This can be renamed to LocationId (only). Using JobOpportunity in the name is redundant since we are in that class already.